render_wrapper.py

```diff
@@ -68,22 +74,18 @@ def __init__(self,
         self.pipeline = DummyPipeline()
 
         self.gaussians = GaussianModel(3)  # 3 is the default sh-degree
-        self.gaussians.load_ply(ply_path)
+        self.gaussians.load_ply(config["ply_path"])
```

Comment:
Does that require two configuration file paths, config_path and ply_path? Are there any other configuration files? And what does 'ply' mean?

Reply:
To initialize the GS_Model class you would only need the config_path... The config.yaml contains the field ply_path, which points to the gaussian splat model (PLY file) relative to the project directory.